test/integration/client/api-tests.js

```diff
@@ -15,6 +15,28 @@ suite.test('pool callback behavior', done => {
   })
 })
 
+suite.test('query timeout', (cb) => {
+  const pool = new pg.Pool({query_timeout: 1000})
+  pool.connect().then((client) => {
+    client.query('SELECT pg_sleep(2)', assert.calls(function (err, result) {
+      assert(err)
+      client.release()
+      pool.end(cb)
+    }))
+  })
+})
+
+suite.test('query timeout', (cb) => {
```

Comment:
is name correct?

Reply:
Right, I'll change it right away. Thanks.

Reply:
@kibertoad done